types/braintree-web/modules/paypal.d.ts

```diff
@@ -63,7 +63,7 @@ export interface PayPal {
      * });
      */
     create(options: { client: Client }): Promise<PayPal>;
-    create(options: { client: Client }, callback: callback): void;
+    create(options: { client: Client }, callback: callback<PayPal>): void;
```

Comment:
Oh nice! So making the callback generic lets us specify the return type now?

Reply:
Yup :)